english/zuluquest/choosequest.md

```diff
@@ -0,0 +1,11 @@
+Congratulations, you are now helping people to decide their destiny! You are a brave person!
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.